src/TrackGenerator.cpp

```diff
@@ -866,6 +863,15 @@ void TrackGenerator::initializeTrackFileDirectory() {
 
 
 /**
+ * @brief Returns the type of ray tracing used for segment formation
+ * @return the segmentation type
```

Comment:
Right now it would read "The only segmentation type currently supported is ``EXPLICIT_2D``". But that doesn't mean a whole lot right now. It would also be incorrect if we added 3D options and forgot to update that comment.